packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -289,6 +291,13 @@ class IconButton extends StatelessWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
+  /// Whether the [splashRadius] should be determined automatically.
+  ///
+  /// This property works only if [splashRadius] is null.
+  /// The [splashRadius] is set based on the [padding] and the [iconSize].
+  /// If false, [Material.defaultSplashRadius] is used.
+  final bool autoImplySplashRadius;
```

Comment:
I don't think we should add this unless the Material Design folks say that's how it should work. The material library is supposed to just have the opinion that matches the material spec, we're trying to avoid adding our own opinions (like how to compute a splash radius). Adding a way to theme the splash radius makes a lot of sense, since that's not adding an opinion other than "things should be configurable" (which the material folks agree with), but adding an algorithm seems like a specific opinion that we'd be adding.

Reply:
Ok , will fix it